charts/icinga-stack/values.yaml

```diff
@@ -365,8 +365,8 @@ icingaweb2:
 
   image:
     repository: icinga/icingaweb2
-    tag: 2.11.4
-    pullPolicy: IfNotPresent
+    tag: 2.12.1
```

Comment:
Same here, please don't bump this version in this PR. If it's needed for Icingaweb-Kubernetes to work, split this PR into 

- one for icinga-kubernetes (the daemon)
- the necessary updates to icingaweb for icingaweb-kubernetes to work

Reply:
See #41.